CHANGELOG.md

```diff
@@ -5,13 +5,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- **cf-buttons:** Convert button links to actual buttons.
```

Comment:
I didn't have `[PATCH]` on this either. I added it but now am not sure if it needs it?